src/js/components/organisms-page/colors/colors.html

```diff
@@ -30,7 +30,9 @@ <h2>{{ title }}</h2>
                             </hit-area>
                         </li>
                     </ul>
-                    <content-link mod="large"><span >{{ labels.whereToBuy }}</span></content-link>
+                    <content-link url="#" mod="large" :label="labels.whereToBuy">
```

Comment:
:hash: ?

Reply:
Hack to make colors work, untill we will actually pick it up in the story